lib/app/utils/validation.rb

```diff
@@ -177,6 +178,11 @@ def self.guess_r4_profile(resource)
         return DEFINITIONS[US_CORE_R4_URIS[:heart_rate]] if observation_contains_code(resource, '8867-4')
 
         return DEFINITIONS[US_CORE_R4_URIS[:body_temperature]] if observation_contains_code(resource, '8310-5')
+
+        return DEFINITIONS[US_CORE_R4_URIS[:vital_signs]] if resource&.category&.first&.coding&.any? { |coding| coding&.code == 'vital-signs' }
```

Comment:
If we do keep vital signs (which I feel like we shouldn't without very strong proof), we should look in any category here, not just the first.  something like `resource&.category&.any{|category| category&.coding&.any{|coding| coding... } }`

Reply:
fixed